examples/with-federated-authentication/src/pages/index.module.css

```diff
@@ -0,0 +1,221 @@
+@font-face {
```

Comment:
this file feels really long for the content of this demo. Are you sure we need all these fonts and all the variants? If not let's trim to only what is used.